### PR TITLE
Deleting a filter on Fritz's SkyPortal deactivates the corresponding filter_v on Kowalski, if such exists

### PR DESCRIPTION
**Pull request by dmitryduev**
Addresses https://github.com/fritz-marshal/fritz/issues/207.

### CODE REVIEW

**Comment by dmitryduev** [comment]
@acrellin [this commit](5af59b8) contains the actual delta with SP's version of `GroupFiltersStreams.jsx`.

**Review by acrellin — APPROVED** [review]

File: extensions/skyportal/static/js/components/GroupFiltersStreams.jsx [hunk 1]
```diff
@@ -153,6 +153,34 @@ const GroupFiltersStreams = ({
...
+                                  } catch (error) {
+                                    throw new Error(`JSON decoding error: ${error}`);
+                                  }
+                                  // exists on Kowalski? deactivate then!
+                                  if (json.status === "success") {
+                                    const result = await dispatch(
+                                      filterVersionActions.editActiveFilterVersion({
+                                        filter_id: filter.id,
+                                        active: false,
+                                      })
```
Comment (acrellin) [inline extensions/skyportal/static/js/components/GroupFiltersStreams.jsx:178]: This submits a request to K to deactivate?

**Review by acrellin — APPROVED** [review]
LGTM, thanks @dmitryduev !

File: extensions/skyportal/static/js/components/GroupFiltersStreams.jsx (hunk 1 above)
Comment (dmitryduev) [reply extensions/skyportal/static/js/components/GroupFiltersStreams.jsx:178]: Yes.

**Pull request merged**